justfile

```diff
@@ -0,0 +1,2 @@
+run:
```

Comment:
@TheEnquirer I swear to god

Reply:
Anyone can run the project now without searching for the command. This should be a standard in all future projects. Certainly will be for mine.

Reply:
Our specific use-case lacks any complex or even existent build recipe. Our build system wrapper does not require a wrapper.